From acd7534926abefd7b4b2fca3ebc361f1e6fa35cb Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 28 Apr 2016 11:00:31 +0200
Subject: [PATCH] mach-imx6sx: Change state of gnss_pwr_e

The state of gnss_pwr_e gpio is now low from the initialsation

Ticket #1133
---
 arch/arm/mach-imx/mach-imx6sx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/mach-imx6sx.c b/arch/arm/mach-imx/mach-imx6sx.c
index 9ff1bdb..256dee6 100755
--- a/arch/arm/mach-imx/mach-imx6sx.c
+++ b/arch/arm/mach-imx/mach-imx6sx.c
@@ -345,7 +345,7 @@ static void imx6sx_gpio_setup_s2(void)
     gpio_request_export(CAN_STB, GPIOF_OUT_INIT_LOW, "can_stb");
     gpio_request_export(CAN_TERM, GPIOF_OUT_INIT_HIGH, "can_term");
     gpio_request_export(GNSS_RES, GPIOF_OUT_INIT_LOW, "gnss_res");
-    gpio_request_export(GNSS_PWR_E, GPIOF_OUT_INIT_HIGH, "gnss_pwr_e");
+    gpio_request_export(GNSS_PWR_E, GPIOF_OUT_INIT_LOW, "gnss_pwr_e");
     gpio_request_export(GNSS_ANT_SEL, GPIOF_OUT_INIT_HIGH, "gnss_ant_sel");
     gpio_request_export(GSM_ANT_SEL, GPIOF_OUT_INIT_HIGH, "gsm_ant_sel");
 	gpio_request_export(UART6_RTS, GPIOF_IN, "uart6_cts");
-- 
1.9.1

